<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>256</width>
      <height>155</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxRecherche">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>571</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Recherche</string>
    </property>
    <widget class="QLineEdit" name="lineEditRecherche">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRecherche">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableViewEmploye">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>511</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>329</y>
      <width>491</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Liste des présences</string>
    </property>
    <widget class="QTableView" name="tableViewListePresence">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>471</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAbsence">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Liste des absences</string>
    </property>
    <widget class="QTableView" name="tableViewListeAbsence">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonConsulter">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consulter </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonModifier">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSupprimer">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConsulter_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consulter absences</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1011</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChargerEmp">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger Employés</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConsulterPresence">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consulter presences</string>
    </property>
   </widget>
   <zorder>calendarWidget</zorder>
   <zorder>groupBoxRecherche</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBoxAbsence</zorder>
   <zorder>pushButtonModifier</zorder>
   <zorder>pushButtonSupprimer</zorder>
   <zorder>pushButtonConsulter_2</zorder>
   <zorder>line</zorder>
   <zorder>pushButtonConsulter</zorder>
   <zorder>pushButtonChargerEmp</zorder>
   <zorder>pushButtonConsulterPresence</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
